maven-surefire-report-plugin/src/main/java/org/apache/maven/plugins/surefire/report/AbstractSurefireReport.java

```diff
@@ -278,17 +281,20 @@ protected I18N getI18N(Locale locale) {
 
         return i18n;
     }
+
     /**
-     * @return The according string for the section.
+     * @return the according string for the section
```

Comment:
Do you want to use the new return Javadoc here (which automatically adds the period at the end)?
If yes, the correct Javadoc to write this would be

```java 
{@return the according string for the section}
```
Which would be transformed to
```
Returns the according string for the section.
```

Reply:
No, per pom.xml we're still using Java 8 here